week02/analyze/Program.cs

```diff
@@ -16,23 +16,23 @@
 
 try {
     MysteryStack2.Run("5 0 /");
-    Console.WriteLine("WRONG: expected ApplicationException: Invalid Case 1!");
+    Console.WriteLine("WRONG: expected ApplicationException: Invalid Case 2!");
```

Comment:
Do we have the `Debug.WriteLine` versus `Console.WriteLine` issue here? Should we do something else here anyway that is more obvious? Maybe an assert or raise a different Exception? (I'm not sure on this. It might be fine as is, but I wanted to note it.)